source/install/self-managed-changelog.md

```diff
@@ -1,14 +1,96 @@
-# Mattermost Changelog
-[Mattermost](https://mattermost.com) is an open source platform for secure collaboration across the entire software development lifecycle. This changelog summarizes updates for all self-hosted versions of Mattermost.
+# Mattermost Self-Hosted Changelog
+[Mattermost](https://mattermost.com) is an open source platform for secure collaboration across the entire software development lifecycle. This changelog summarizes updates for the latest self-hosted versions of Mattermost.
 
-Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
+See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v5.37. 
 
 Latest Mattermost Releases:
-- [Release v6.2 - Feature Release](#release-v6.2-feature-release)
-- [Release v6.1 - Feature Release](#release-v6.1-feature-release)
-- [Release v6.0 - Feature Release](#release-v6.0-feature-release)
-- [Release v5.39 - Quality Release](#release-v5-39-quality-release)
-- [Release v5.37 - Extended Support Release](#release-v5-37-extended-support-release)
+- [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
+- [Release v6.2 - Feature Release](#release-v6-2-feature-release)
+- [Release v6.1 - Feature Release](#release-v6-1-feature-release)
+- [Release v6.0 - Feature Release](#release-v6-0-feature-release)
+
+## Release v6.3 - [Extended Support Release](https://docs.mattermost.com/upgrade/release-definitions.html#extended-support-release-esr)
+
+**Release Day: 2022-01-16**
+
+### Important Upgrade Notes
+ - An asynchronous job was added to fix an issue with Collapsed Reply Threads that resulted in unexpected unread channels. The job may result in an increase in server and database resource usage. Depending on the size of your Mattermost database, this job could take several hours to complete. In our internal testing it took about 3 hours to check and fix a ``ChannelMembership`` table with about 4 million rows in a PostgreSQL database.
```

Comment:
```suggestion
 - Mattermost v6.3 adds an asynchronous job to fix an issue that resulted in channels with join/leave messages being unexpectedly marked as unread when enabling Collapsed Reply Threads. After the upgrade, the job is started automatically when the server restarts and the job executes in the background so the Mattermost server will not experience downtime. We recommend upgrading during off-peak hours as you may experience higher than average server and database resource utilization while the job is in progress. In our testing, the job takes approximately 3 hours per ~4 million rows in the ChannelMembership table using a PostgreSQL database.

You may check the progress of the job by executing this database query: ``select * from Jobs where type = 'fix_channel_unreads_for_crt' order by lastactivityat desc;``. The data column will display the number of channel memberships fixed (BadChannelMembershipsFixed), the number of channel memberships checked (TotalChannelMembershipsChecked), and the current Channel and User ID pair that is being processed.

If you are experiencing severely high server and database resource utilization, you may kill the job. First update the Systems table to stop the job from restarting
``INSERT INTO Systems VALUES ('fix_crt_channel_unreads', true);``

Then restart the Mattermost application to kill the existing job.

Once the job is complete, it will not run again and resource usage will return to average levels. Restarting the Mattermost server will automatically reschedule the job to run after a few minutes, if the ``fix_crt_channel_unreads`` key is not set to ``true`` in the Systems table. If that key is set, then deleting it from the Systems table will reschedule the job after a few minutes.
```